test/e2e/density.go

```diff
@@ -186,6 +209,108 @@ var _ = Describe("Density", func() {
 			// Tune the threshold for allowed failures.
 			badEvents := BadEvents(events)
 			Expect(badEvents).NotTo(BeNumerically(">", int(math.Floor(0.01*float64(totalPods)))))
+
+			if itArg.runLatencyTest {
+				Logf("Schedling additional Pods to measure startup latencies")
+				// Use service to assure even spread of pods between nodes
```

Comment:
I don't really know how the scheduler works right now, but I was seeing lack of spreading of plain Pods, hence the service which seems to help a bit (anectdata from few small runs - might be an accident). I don't fully understand second sentence - are you saying that adding service will force the spread, or that it won't change anything?